bucket/audmes.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "20211104",
+    "description": "AUDio MEasurement System - PC based Oscilloscope and Spectrum analyzer using sound card",
+    "homepage": "https://sourceforge.net/projects/audmes/",
+    "license": "GPLv2",
+    "url": "https://phoenixnap.dl.sourceforge.net/project/audmes/audmes-win32%20binary/2021-11-04/AudMeS-20211104-win32.zip",
+    "hash": "sha256:ca6da925ae4fdd63a2cc0c92901923c2434312a4617170a50db0858960968a73",
```

```suggestion
    "hash": "ca6da925ae4fdd63a2cc0c92901923c2434312a4617170a50db0858960968a73",
```
`sha256:` is actually not needed because it is the default format.